src/addValidator.ts

```diff
@@ -63,7 +63,7 @@ export async function getUnsignedAddValidator(
   }
 }
 
-async function getAddValidatorParams(
+export async function getAddValidatorParams(

```

Comment:
please add documentation for all the exported function, interfaces, constants and enums